test/e2e/pivoting.go

```diff
@@ -83,6 +83,22 @@ func pivoting(ctx context.Context, inputGetter func() PivotingInput) {
 		fetchContainerLogs(&ironicContainers, input.ArtifactFolder, input.E2EConfig.GetVariable("CONTAINER_RUNTIME"))
 	}
 
+	By("Fetch manifest for ephemeral cluster")
+	path := filepath.Join(os.Getenv("CAPM3PATH"), "scripts")
+	cmd := exec.Command("./fetch_manifests.sh") // #nosec G204:gosec
+	cmd.Dir = path
+	outputPipe, _ := cmd.StdoutPipe()
+	errorPipe, _ := cmd.StderrPipe()
```

Comment:
would prefer to use `cmd.CombinedOutput()`

Reply:
We have done similar in other command run in e2e. So we can improve them in future.